cypress/e2e/staff_admin/prefix.test.ts

```diff
@@ -8,11 +8,17 @@ describe('Admin: Prefix', () => {
   before(function () {
     cy.login(Cypress.env('staff_admin_username'), Cypress.env('staff_admin_password'));
     cy.setCookie('_consent', 'true');
+    cy.wait(waitTime2);
   })
 
   beforeEach(() => {
-    Cypress.Cookies.preserveOnce('_fabrica', '_jwt', '_consent');
-    cy.wait(waitTime2);
+    // TBD - set up test environment
```

Comment:
I've seen this a couple of times.   Is this still TBD in all of these cases?  Cypress no longer needs the cookies preserved?

Reply:
That should be gone.  It should not be used at all.  Will fix it and re-run all tests.